### PR TITLE
feat: implement listing of comments in a thread

### PR DESCRIPTION
**Pull request by lucasmori**

### CODE REVIEW

File: src/main/java/masecla/reddit4j/objects/adapters/RedditCommentRepliesAdapter.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import masecla.reddit4j.objects.RedditComment;
+import masecla.reddit4j.objects.RedditData;
+import masecla.reddit4j.objects.RedditListing;
+
+import java.io.IOException;
+
+public class RedditCommentRepliesAdapter extends TypeAdapter<RedditData<RedditListing<RedditData<RedditComment>>>> {
+    @Override
+    public void write(JsonWriter out, RedditData<RedditListing<RedditData<RedditComment>>> value) throws IOException {
+        System.out.println("Write");
```
Comment (masecla22) [inline src/main/java/masecla/reddit4j/objects/adapters/RedditCommentRepliesAdapter.java:17]: This left in here by accident?

**Review by masecla22 — COMMENTED** [review]
<3

File: src/main/java/masecla/reddit4j/objects/adapters/RedditCommentRepliesAdapter.java (hunk 1 above)
Comment (lucasmori) [reply src/main/java/masecla/reddit4j/objects/adapters/RedditCommentRepliesAdapter.java:17]: Oh, this could be remove.

Comment (lucasmori) [inline src/main/java/masecla/reddit4j/objects/adapters/RedditCommentRepliesAdapter.java:17]:
```suggestion
```

**Pull request merged**